source.go

```diff
@@ -222,6 +273,45 @@ type SourceMandate struct {
 	URL                string                          `json:"url"`
 }
 
+// SourceSourceOrderShippingAddress describes the Address for source orders on
+// sources.
+type SourceSourceOrderShippingAddress struct {
+	City       string `json:"city"`
+	Country    string `json:"country"`
+	Line1      string `json:"line1"`
+	Line2      string `json:"line2"`
+	PostalCode string `json:"postal_code"`
+	State      string `json:"state"`
+}
+
+// SourceSourceOrderItems describes the items on source orders for sources.
+type SourceSourceOrderItems struct {
+	Amount      int64                     `json:"amount"`
+	Currency    Currency                  `json:"currency"`
+	Description string                    `json:"description"`
+	Quantity    int64                     `json:"quantity"`
+	Type        SourceSourceOrderItemType `json:"type"`
+}
+
+// SourceSourceOrderShipping describes the shipping options for source orders on
+// sources.
+type SourceSourceOrderShipping struct {
```

Comment:
This should re-use the normal address fields that exist elsewhere. That's what we do in Java where it uses `ShippingDetails`: https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Source.java#L877

So here in go you want to use `Shipping` and `ShippingParams` I thiiiiink